dotnet/src/Extensions/PromptTemplates.Handlebars/Helpers/KernelHelpers/KernelFunctionHelpers.cs

```diff
@@ -22,18 +23,20 @@ internal static class KernelFunctionHelpers
     /// <param name="handlebarsInstance">The <see cref="IHandlebars"/>-context.</param>
     /// <param name="kernel">Kernel instance.</param>
     /// <param name="executionContext">Kernel arguments maintained as the executing context.</param>
+    /// <param name="promptModel">The associated prompt template configuration.</param>
     /// <param name="nameDelimiter">The character used to delimit the plugin name and function name in a Handlebars template.</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     public static void Register(
         IHandlebars handlebarsInstance,
         Kernel kernel,
         KernelArguments executionContext,
+        PromptTemplateConfig promptModel,
```

Comment:
nit: `promptConfig`/`promptTemplateConfig` instead of `promptModel`? We can also pass just `bool allowUnsafeContent`, since it's the only property from `PromptTemplateConfig` object that is used here.